### ISSUE TITLE
Build error on Python3.6

### ISSUE THREAD

**Issue by geekodour** [pythonindia/inpycon2017#46]
It works alright on python2.7

**Comment by ashwamegh** [comment]
@geekodour Please post a screenshot here!

**Comment by CuriousLearner** [comment]
@geekodour Build error does not mean anything to us. Can you please show the error you get?

**Comment by geekodour** [comment]
![image](http://i.imgur.com/ZH82WB1.png)

I am using python2.7 now..

**Comment by aakashbajaj** [comment]
It seems like your python default has been set to 3.6.1
Set it back to python2.7 and check again

**Comment by aradhyamathur** [comment]
check the  `__init__.py` of easywatch in the env change it to `from .easywatch import watch` that could be a workaround. It seems to be a bug in older easywatch on python3.
You can refer to this for further details https://github.com/Ceasar/staticjinja/issues/56

**Comment by SanketDG** [comment]
Yup, looks like the maintainer has fixed it, but not released the package yet.

The only feasible solution is to add the github mirror to requirements:
```
git+git://github.com/Ceasar/easywatch.git
```

For users who already have easywatch, this would mean to do a `pip install --upgrade -r requirements.txt` to initiate a force upgrade of easywatch to the github mirror instead of pypi.

Rest can do `pip install -r requirements.txt`
